src/components/TaskFilterButtons/index.jsx

```diff
@@ -1,21 +1,52 @@
 import React from "react";
 import { useState } from "react";
 import "./TaskFilterButtons.css";
+
 function TaskFilterButtons({ setFilterStatus, tasks, handleClearTask }) {
   const itemLeft = tasks.length;
 
+  const [currFilter, setCurrFilter] = useState("");
+
+  const changeFilter = (filter) => {
+    setCurrFilter(filter);
+    setFilterStatus(filter);
+  };
+
+  const statusFilter = {
+    all: "All",
+    complete: "Complete",
+    active: "Active",
+  };

```

Comment:
change it to 

const STATUS_FILTER = {
    ALL: "All",
    COMPLETE: "Complete",
    ACTIVE: "Active",
  };
  
  and move it before the definition of the component

That is the naming conventions it is normally used when a constant value is defined (I am not talking about const)

Also, if you have a constant value like that, it is preferable to be outside of the component definition to improve the performance and speed of the component